CMSIS/Core/Include/cmsis_compiler.h

```diff
@@ -53,6 +53,15 @@
  */
 #elif defined ( __ICCARM__ )
 
+  /* CMSIS compiler control architecture macros */
```

Comment:
I updated it now we dont redefine if already defined. Or we can use similar what they use: ``__ARM_ARCH`` ?